src/dataframe/dataframe.jl

```diff
@@ -33,7 +33,7 @@ DataFrame(ds::Vector{AbstractDict})
 * `column_eltypes` : elemental type of each column
 * `categorical` : `Vector{Bool}` indicating which columns should be converted to
                   `CategoricalVector`
-* `ds` : a vector of Associatives
+* `ds` : `AbstractDict`
```

Comment:
Specify "of columns" or something like that?

Reply:
fixed